src/data/adventures.txt

```diff
@@ -7,696 +7,696 @@ Campground	adventure=543	Env: outdoor	Your Mushroom Garden
 
 # Clan areas
 
```

Comment:
Does this change the version of the file?  Should it?  If so we should also look at tests since I know some tests have a hardwired version rather than just reading the latest/current version.

Reply:
version number should change if you add a new tab delimited field.
Programs that read the file presumably are coded to work with a specific version.
If tests are coded to work with a specific version, if the file format changes, they should/must be updated.